sections/aws.zsh

```diff
@@ -25,13 +25,16 @@ spaceship_aws() {
   # Check if the AWS-cli is installed
   spaceship::exists aws || return
 
+  # Interpolate Native or aws-vualt profile
+  SPACESHIP_AWS_PROFILE="${AWS_PROFILE:-"$AWS_VAULT"}"
```

Comment:
Local variables shouldn't prepend `SPACESHIP_` and it has to be named in lowercase.

Reply:
Sounds good, I did not notice `local`s were actually used in other sections. Updated as a fixup till we are ready to merge this, so its easier to review.
I also renamed the var `profile` to avoid confusion with `AWS_PROFILE`, let me know your thoughts.